### PR TITLE
Feature/room to moderator

### PR DESCRIPTION
**Pull request by CPower1248**
Created room-to-moderator table, added routes and models for room-to-moderator endpoints. Updated documentation to reflect new endpoints. Created middleware folder to abstract DRY code.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,6 +25,12 @@ const FRONTEND_URL =
   process.env.FRONTEND_DEPLOYED_URL || 'http://localhost:3000';
 const PORT = process.env.PORT || 5000;
 
+app.all('/*', function(req, res, next) {
```
Comment (brennanlawrence) [inline index.js]: Looks Great! Though right here you seem to be adding duplicate code. app.all("/*)etc... 

Also, we should probably remove the app.all() beneath app.use(helmet()). That is totally my bad, I should have removed that.

**Review by brennanlawrence — CHANGES_REQUESTED** [review]
Looks great, just one or two pieces of duplicate code to remove.

File: index.js (hunk 1 above)
Comment (CPower1248) [reply index.js]: Just removed the duplicate. I'll remove the app.all() as well

**Review by brennanlawrence — CHANGES_REQUESTED** [review]
Hey sorry, the other app.all("/*") also duplicates functionality that is in app.use below helmet()

**Review by brennanlawrence — APPROVED** [review]

**Review by mmitch2958 — APPROVED** [review]
Looks Great, Love the middleware !

**Pull request merged**